addon/templates/components/polaris-pagination.hbs

```diff
@@ -1,39 +1,39 @@
-<button
-  data-test-prev-btn="true"
-  class="Polaris-Pagination__Button"
-  aria-label="Previous"
-  disabled={{isPreviousDisabled}}
-  onclick={{action onPrevious}}
-  onmouseup={{handleMouseUpByBlurring}}
+<nav
+  aria-label={{this.accessibilityLabel}}
+  class="Polaris-Pagination {{if @plain "Polaris-Pagination--plain"}}"
+  ...attributes
 >
-  {{polaris-icon source="arrow-left"}}
-</button>
+  <button
+    data-test-prev-btn="true"
+    class="Polaris-Pagination__Button"
+    aria-label="Previous"
+    disabled={{this.isPreviousDisabled}}
+    {{on "click" (fn (or @onPrevious this.onPrevious))}}
```

Comment:
nit-pick: I think we can get away without `fn` helper usage....you'd generally need it in case you want to curry any arguments to the action

Reply:
Yeah it's not needed, I remember we used to explicitly use the `action` helper sometimes even when it wasn't required so I wasn't sure if we wanted to follow that practice with `fn` or not. I can stop doing this if we don't think it's necessary anymore.

Reply:
yeah, we won't need it anymore 🙌